src/types/venues.ts

```diff
@@ -236,6 +235,7 @@ export interface PartyMapVenue extends BaseVenue {
   admin_password?: string;
   owners: string[];
   rooms?: Room[];
+  events: WithId<VenueEvent>[];
```

Comment:
1.  why it was added to PartyMap when BaseVenue has already events?
2. I believe we should better handle events separately and not mutate the object by adding events there, as we don't store the information about events in the firestore-venue-object.